configs/dh-defaults.prop

```diff
@@ -45,123 +14,41 @@ measurement.log_and_reset=false
 # TODO: all of the created threads are waiting on RPCs or locks.
 #
 NIO.driver.workTimeout=100
-NIO.driver.initialThreadCount=2
-NIO.driver.maxThreadCount=400
+NIO.driver.initialThreadCount=4
+NIO.driver.maxThreadCount=16
 NIO.driver.useFast=true
 
-#
-# NIO wire-lag clock
-#
-NIO.wireLagClock.native=false
-
-#
-# Default envelope handler for WClient instances
-#
-Comm.fatalErrorHandlerFactoryClass=io.deephaven.console.utils.ProcessEnvironmentRedirectFactory
-
-##### CompilerTools Settings #####
-
-CompilerTools.logEnabledDefault=false
-
 QueryLibrary.defaultPackageImportList=default_package_imports.txt
 QueryLibrary.defaultClassImportList=default_class_imports.txt
 QueryLibrary.defaultStaticImportList=default_static_imports.txt
 
-########## Deephaven Groovy and Python Session defaults ##########
-
-GroovyDeephavenSession.defaultScriptPath=<devroot>
-GroovyDeephavenSession.initScripts=core/deephaven_core_utils.groovy
-
-PythonDeephavenSession.defaultScriptPath=<devroot>
-PythonDeephavenSession.initScripts=core/deephaven_jpy_init.py
-
-WorkerPythonEnvironment.defaultScriptPath=<devroot>
-
-########## Deephaven UpdatePerformanceTracker Properties ##########
-UpdatePerformanceTracker.reportingMode=LISTENER_ONLY
-UpdatePerformanceTracker.reportIntervalMillis=60000
-
-########## Deephaven QueryPerformanceRecorder Properties ##########
-QueryPerformanceRecorder.packageFilter.internal=defaultPackageFilters.qpr
+StringUtils.cacheSize=0
 
-########## Deephaven Local Data Refresh Properties ##########
-TableDataRefreshService.tableLocationsRefreshMillis=10000
-TableDataRefreshService.tableSizeRefreshMillis=1000
+TrackedFileHandleFactory.maxOpenFiles=1024
 
-# Defines the directory for pidfiles
-pidFileDirectory=/etc/deephaven/run
```

Comment:
Can you restore this, but as `/var/run/deephaven` for now? If someone tries to use PidFileUtil it will blow up.

Reply:
You know, I think I'm wrong about this. It would probably be better to blow up and make someone consciously choose a value.

Reply:
If the server doesn't use it though, should we reference it here? Especially if we are set up to run the server instances as plain user processes? We would otherwise need to also add logic to create the directory structure, and probably as root (and "not at all" on windows).

Relative dir to execution okay?